migration_steps/prepare_source_data/counts_verification/count_existing_pc1.py

```diff
@@ -0,0 +1,68 @@
+import os
```

Comment:
Could we have a single script for both of these as they do practically the same thing? Or one script which runs both steps?

Reply:
Decent point, I'll look at combining the scripts

Reply:
I've combined scripts 1 and 2 into a longer 'pre-migrate counts' script, but it makes logical sense I think for script 3 to stay with the other post-migrate stuff in the directory that comprises the validation docker container